cedar-policy/src/api.rs

```diff
@@ -1912,6 +1918,7 @@ impl PolicySet {
 
     /// Remove a `Template` from the `PolicySet`.
     /// If any policy is linked to the template, this will error
+    /// If the policy is not a template this will error
```

Comment:
```suggestion
    /// Remove a `Template` from the `PolicySet`
    ///
    /// If any policy is linked to the template, this will error.
    /// If the policy is not a template, this will also error.
```